wgpu/src/lib.rs

```diff
@@ -4531,3 +4581,51 @@ impl Display for Error {
         }
     }
 }
+
+use send_sync::*;
+
+mod send_sync {
+    use std::any::Any;
+    use std::fmt;
+
+    #[cfg(not(target_arch = "wasm32"))]
+    pub trait MaybeSend: Send {}
```

Comment:
We probably should call it something other than `MaybeSend`, as users will need to use it too.`WasmNotSend` or so? Suggestions welcome

Reply:
I thought about this but honestly can't come up with a good name.
Also played around with re-exporting `Send` and `NoSend` under the same name and hoping that `rustdoc` would show it under it's actual name, but it didn't.

I will just use `WasmNotSend`.

Reply:
Done.